packages/openapi/README.md

```diff
@@ -0,0 +1,17 @@
+# OpenAPI 3.1 Validator
```

Comment:
I think we're going to want to use this for:
-  https://github.com/interledgerjs/interledgerjs/issues/283

If so, should this exist somewhere outside the Rafiki monorepo?